lib/node_modules/@stdlib/array/ones-like/docs/types/index.d.ts

```diff
@@ -648,10 +46,10 @@ declare function onesLike( x: Uint8ClampedArray, dtype?: DataType ): Uint8Clampe
 * var y = onesLike( x );
 * // returns [ 1, 1 ]
 */
-declare function onesLike( x: Array<any>, dtype?: DataType ): Array<number>;
+declare function onesLike<T extends AnyArray>( x: T ): T;
```

Comment:
How does this work for generic arrays? Shouldn't we always be returning an `Array<number>`?

Reply:
Addressed.